src/sentry/static/sentry/app/views/eventsV2/eventDetails/pagination.tsx

```diff
@@ -64,75 +64,62 @@ const Pagination = (props: Props) => {
   const links = buildTargets(event, eventView, organization);
 
   return (
-    <Wrapper>
-      <ShadowBox>
-        <StyledLink
-          to={links.oldest}
-          disabled={links.previous === null || links.oldest === null}
-        >
-          <InlineSvg src="icon-prev" size="14px" />
-        </StyledLink>
-        <StyledLink
-          data-test-id="older-event"
-          to={links.previous}
-          disabled={links.previous === null}
-        >
-          {t('Older Event')}
-        </StyledLink>
-        <StyledLink
-          data-test-id="newer-event"
-          to={links.next}
-          disabled={links.next === null}
-        >
-          {t('Newer Event')}
-        </StyledLink>
-        <StyledLink
-          to={links.latest}
-          disabled={links.next === null || links.latest === null}
-          isLast
-        >
-          <InlineSvg src="icon-next" size="14px" />
-        </StyledLink>
-      </ShadowBox>
-    </Wrapper>
+    <Paginator>
+      <StyledIconLink
+        to={links.oldest}
+        disabled={links.previous === null || links.oldest === null}
+      >
+        <InlineSvg src="icon-prev" size="14px" />
+      </StyledIconLink>
+      <StyledTextLink
+        data-test-id="older-event"
+        to={links.previous}
+        disabled={links.previous === null}
+      >
+        {t('Older Event')}
+      </StyledTextLink>
+      <StyledTextLink
+        data-test-id="newer-event"
+        to={links.next}
+        disabled={links.next === null}
+      >
+        {t('Newer Event')}
+      </StyledTextLink>
+      <StyledIconLink
+        to={links.latest}
+        disabled={links.next === null || links.latest === null}
+        isLast
+      >
+        <InlineSvg src="icon-next" size="14px" />
+      </StyledIconLink>
+    </Paginator>
   );
 };
 
-const StyledLink = styled(Link, {
-  shouldForwardProp: isPropValid,
-})<{theme: any; disabled: boolean; isLast: boolean}>`
+const StyledTextLink = styled(Link, {shouldForwardProp: isPropValid})<{
+  theme: any;
+  disabled: boolean;
+  isLast: boolean;
+}>`
   color: ${p => (p.disabled ? p.theme.disabled : p.theme.gray3)};
   font-size: ${p => p.theme.fontSizeMedium};
-  text-align: center;
   padding: ${space(0.5)} ${space(1.5)};
+  text-align: center;
+  flex-grow: 2;
   ${p => (p.isLast ? '' : `border-right: 1px solid ${p.theme.borderDark};`)}
   ${p => (p.disabled ? 'pointer-events: none;' : '')}
-
-  @media (max-width: ${p => p.theme.breakpoints[0]}) {
-    flex-grow: 1;
-  }
 `;
 
-const Wrapper = styled('div')`
-  display: flex;
-
-  @media (max-width: ${p => p.theme.breakpoints[0]}) {
-    width: 100%;
-  }
+const StyledIconLink = styled(StyledTextLink)`
```

Comment:
Need this here to address the errors from the tests:

```suggestion
const StyledIconLink = styled(StyledTextLink, {shouldForwardProp: isPropValid})`
```

Reply:
For context, emotion wasn't forwarding some props like `isLast` to the wrapped component; and was only applying them to `StyledIconLink`, which was the cause for the errors in the tests. 

Some more info on this https://emotion.sh/docs/styled#customizing-prop-forwarding